SQLqueries.txt

```diff
@@ -0,0 +1,48 @@
+Queries:
+1. display all members of a particular organizing sub-committee  (allow the user to choose the sub-committee from a drop down menu).
+
+SELECT *
+FROM IsMember
```

Comment:
We don't need this query to be a join do we? I think we can just do a `WHERE` clause with `subcommitteeName = someCommitteeName'

Reply:
You're right that makes a lot more sense.